### ISSUE TITLE
Miner does not start after restarting the computer

### ISSUE THREAD

**Issue by RHGF44** [UnamSanctam/SilentCryptoMiner#1632]
There is a problem. The miner does not start after restarting the computer. until you restart manually. What to do?

**Comment by UnamSanctam** [comment]
Do you have "Startup" enabled? And if so can you find the entry inside the "Task Scheduler" (admin) or registry (non-admin)?

**Comment by RHGF44** [comment]
Of course, autoload is enabled. I can't view the registry. How to find an entry in the registry?

**Comment by UnamSanctam** [comment]
Do you install it as administrator or a normal user? And do you have "Run as System" enabled or not? If it's installed as administrator then it will be inside the Task Scheduler. If you installed it as a user then you need to find it in the registry, so open regedit.exe and go to this path `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Run` and if it's correctly installed there then there should be an entry there with the "Entry Name" from inside the "Startup" tab when you built the miner.

**Comment by RHGF44** [comment]
If I don't have autoload in a miner that works, how can I replace a miner that was pre-built with autoload?

**Comment by UnamSanctam** [comment]
If the installation was not successful then you can just install the new miner, or you can run the uninstaller of the old miner first to be safe.

**Comment by RHGF44** [comment]
Thank you.
Is it safer to do autoload from detection in the scheduler or registry?

**Comment by UnamSanctam** [comment]
The Task Scheduler usually.

**Comment by RHGF44** [comment]
Can I use spaces in miner name?

**Comment by UnamSanctam** [comment]
Yes that should be fine.
